spec/lib/pg_search/multisearch/rebuilder_spec.rb

```diff
@@ -223,6 +223,31 @@ def foo
             expect(record.pg_search_document).to be_present
           end
         end
+
+        context "when only addicional_attributes is set" do
```

Comment:
Fix spelling addicional -> additional

Reply:
I'm so Portuguese sometimes :P

Reply:
I made a worse job with the name of this branch to be honest :)

Reply:
Yeah no worries :) I am going to see what I did wrong when trying to get the spec to fail...